### ISSUE TITLE
add typetuner support

### ISSUE THREAD

**Issue by mhosken** [silnrsi/smith#11]
smith has the specification for a typetuner= parameter in a font creation, but it currently does nothing. We need to write the code for this. But what does this actually do?

**Comment by bobh0303** [comment]
For now, until we figure out how to extract some aliases, we assume the typetuner xml file is manually maintained. Thus the command to add tuner information is:
```
typetuner -o out.ttf add typetuner.xml in.ttf
```
I'm sure that isn't the syntax you need to pick up the two input files and specify the output, but should be enough info for you to build the rules into smith.

**Comment by bobh0303** [comment]
In case it is of help to the person implementing this, an equivalent post-build process we're using in LateefGR is:
```
font(target = process('LateefGR-Regular.ttf', cmd('${TYPETUNER} -o ${TGT} add ${SRC} ${DEP}', 'source/typetuner.xml')),
    ...
```

**Comment by mhosken** [comment]
Being addressed in #48
